CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/undo/UndoGraphEditState.java

```diff
@@ -138,6 +136,7 @@ public UndoGraphEditState(DataInputStream in) throws Exception {
                 objectClass = Class.forName(objectName);
                 classMap.put(objectClassIndex, objectClass);
             }
+            classMap.forEach((k, v) -> objectMap.put(v, k));
```

Comment:
@cygnus-x-1 i don't get why we need to do this. I tested undo and redo and it works fine in this PR but doing a `forEach` inside the `for` loop is going to be a performance hit and feels unnecessary. `objectMap` gets added to in `addObject()`.

Reply:
`addObject` is irrelevant here as I believe the intention here is to construct the class from a previous state. I get what I was going for here, as objectMap was not being initialized properly, but I'm not sure why I did it this way when I could have just added each entry to `objectMap` alongside `classMap`. This is a bug (that we didn't notice because it's not used), but I'll update it to a more efficient implementation.